### PR TITLE
Added ability to convert inches per second into ticks per 10ms. Added…

### PR DESCRIPTION
**Pull request by SethNguyenXBOT**
… test to see if new ability works.

### CODE REVIEW

File: Competition2017/src/competition/subsystems/drive/DriveSubsystem.java [hunk 1]
```diff
@@ -128,7 +128,7 @@ private void ensurePowerModeForDrive() {
         ensureModeForTalon(rightDrive, TalonControlMode.PercentVbus);
     }
 
-    public void tankDriveVelocityPid(double leftPower, double rightPower) {
+    public void tankDriveVelocityPercentMax(double leftPower, double rightPower) {
```
Comment (JohnGilb) [inline Competition2017/src/competition/subsystems/drive/DriveSubsystem.java]: rename these arguments to something like `leftVelocityPercent` and `rightVelocityPercent`

File: Competition2017/src/competition/subsystems/drive/DriveSubsystem.java [hunk 2]
```diff
@@ -145,6 +145,20 @@ public void tankDriveVelocityPid(double leftPower, double rightPower) {
...
     
+    public void tankDriveVelocityInchesPerSec(double leftInchesPerSec, double rightInchesPerSec) {
+        // TODO: Move parameter updates to something more consistent
+        ensureSpeedModeForDrive();
+        
+        updateMotorConfig(leftDrive);
+        updateMotorConfig(rightDrive);
+
+        //Converting inches per second to ticks per 10ms
+        leftDrive.set(convertInchesToTicks(leftInchesPerSec)/100.0);
```
Comment (JohnGilb) [inline Competition2017/src/competition/subsystems/drive/DriveSubsystem.java]: you say 10ms in the comment above, but the code here is dividing by 100ms. Which is right?

Comment (WasabiFan) [reply Competition2017/src/competition/subsystems/drive/DriveSubsystem.java]: `per 1000ms` / `100` = `per 10 ms`

File: Competition2017/src/competition/subsystems/drive/DriveSubsystem.java [hunk 3]
```diff
@@ -145,6 +145,20 @@ public void tankDriveVelocityPid(double leftPower, double rightPower) {
...
     }
     
+    public void tankDriveVelocityInchesPerSec(double leftInchesPerSec, double rightInchesPerSec) {
+        // TODO: Move parameter updates to something more consistent
+        ensureSpeedModeForDrive();
+        
+        updateMotorConfig(leftDrive);
+        updateMotorConfig(rightDrive);
+
+        //Converting inches per second to ticks per 10ms
```
Comment (aschokking) [inline Competition2017/src/competition/subsystems/drive/DriveSubsystem.java]: Nit, space after `// `

File: Competition2017/tests/competition/subsystems/drive/DriveSubsystemTest.java [hunk 4]
```diff
@@ -77,5 +77,29 @@ public void lowPowerInputTankDriveTest(){
         drive.tankDrivePowerMode(0.01, -0.01);
         verifyDriveSetpoints(0.01, -0.01);
     }
-}
+    
+    public void testUpdatePeriodicDataTest(){
```
Comment (aschokking) [inline Competition2017/tests/competition/subsystems/drive/DriveSubsystemTest.java]: This was deleted in another commit, probably a bad merge left it here, safe to delete again from your branch.

**Review by aschokking — CHANGES_REQUESTED** [review]

**Review by aschokking — APPROVED** [review]

**Pull request merged**